skopt/optimizer/optimizer.py

```diff
@@ -176,7 +186,101 @@ def _check_arguments(self, base_estimator, n_random_starts, acq_optimizer):
                              "'sampling', got {0}".format(acq_optimizer))
         self.acq_optimizer = acq_optimizer
 
-    def ask(self):
+    def copy(self, copy_random_state=True):
+        """Creates a shallow copy of optimizer."""
```

Comment:
More docs :) Also "Create ..." instead of "Creates ..." https://www.python.org/dev/peps/pep-0257/ has a few more suggestions on wording (even as a native speaker I keep checking that I follow it)